src/wallet/rpcdump.cpp

```diff
@@ -474,15 +474,7 @@ UniValue dumpwallet_impl(const UniValue& params, bool fHelp, bool fDumpZKeys)
     } catch (const std::runtime_error& e) {
         throw JSONRPCError(RPC_INTERNAL_ERROR, e.what());
     }
-    if (exportdir.empty()) {
```

Comment:
A couple of issues I have with this are:
- We're no longer "sanitizing" the input. This can lead to some undesirable behavior by the code.
- If exportdir is set as an empty string, would this save to `/ + first param`? If so, we should still check if exportdir is empty. If it is, we should not be prepending the `/`.